src/main/java/nl/knaw/dans/easy/DataciteService.java

```diff
@@ -88,12 +90,38 @@ private String postDoi(String content) throws DataciteServiceException {
         }
     }
 
+    public boolean getDoi(String doi) throws DataciteServiceException {
```

Comment:
given that you return a `boolean`, I would say this method should be called something like `doiExists`.
But I'm not sure what the specific purpose of this method will be (what it needs to return in order to be useful).

Reply:
Food for thought. Ill will discuss this with joke later today